Lombiq.Tests.UI/Services/UITestExecutionSession.cs

```diff
@@ -109,15 +111,19 @@ public async Task<bool> ExecuteAsync(int retryCount, string dumpRootPath)
 
             if (ex is SetupFailedFastException) throw;
 
-            await CreateFailureDumpAsync(ex, dumpRootPath, retryCount, failureDumpContainer);
+            await CreateTestDumpAsync(
+                dumpRootPath,
+                retryCount,
+                testDumpContainer,
+                dumpContainerPath => FailureTestDumpProcessAsync(dumpContainerPath, ex));
 
             if (_context?.IsFinalTry == true || retryCount >= _configuration.MaxRetryCount)
             {
                 var dumpFolderAbsolutePath = Path.Combine(AppContext.BaseDirectory, dumpRootPath);
 
                 _testOutputHelper.WriteLineTimestampedAndDebug(
                     "The test was attempted {0} time(s) and won't be retried anymore. You can see more details " +
-                        "on why it's failing in the FailureDumps folder: {1}",
+                        "on why it's failing in the TestDumps folder: {1}",
```

Comment:
It's not necessary to re-state the folder name, it's already in the `dumpFolderAbsolutePath`.
```suggestion
                        "on why it's failing in the test dump folder: {1}",
```

Reply:
Indeed.